test/FunctionalTests/Microsoft.OData.Core.Tests/JsonLight/ODataJsonLightEntryAndFeedSerializerUndecalredTests.cs

```diff
@@ -1,4 +1,6 @@
-﻿namespace Microsoft.Test.OData.TDD.Tests.Writer.JsonLight
+﻿using Microsoft.OData;
+

```

Comment:
Okay. I didn't want to propagate the spelling error in the copyright, so I will also rename the file.

---
In reply to: [128103905](https://github.com/OData/odata.net/pull/894#discussion_r128103905) [](ancestors = 128103905)